hooks/before_prepare/download_translation.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+'use strict'
+
+var child_process = require('child_process')
```

Comment:
IIUC, at the end of this hook, the translations will be in the local `locales` directory

Reply:
Yes, and if the directory already exists, it will just pull the repo instead of cloning it.